pobieracz_danych_gugik.py

```diff
@@ -1325,37 +1325,56 @@ def model3d_selected_powiat_btn_clicked(self):
         elif self.dockwidget.model3d_lod1_rdbtn.isChecked() and self.dockwidget.model3d_lod2_rdbtn.isChecked():
             standard = ["LOD1", "LOD2"]
         elif not self.dockwidget.model3d_lod1_rdbtn.isChecked() and not self.dockwidget.model3d_lod2_rdbtn.isChecked():
-            msgbox = QMessageBox(QMessageBox.Information, "Ostrzeżenie:",
-                                 f"Nie wybrano standardu")
+            msgbox = QMessageBox(QMessageBox.Information, "Ostrzeżenie:", "Nie wybrano standardu")
             msgbox.exec_()
             return False
 
         od_data_text = self.dockwidget.model3d_dateEdit_comboBox_1.currentText()
         do_data_text = self.dockwidget.model3d_dateEdit_comboBox_2.currentText()
 
-        try:
-            od_data = int(od_data_text)
-        except ValueError:
-            msgbox = QMessageBox(QMessageBox.Warning, "Błąd", "Nieprawidłowa data początkowa.")
-            msgbox.exec_()
-            return False   
-        try:
-            do_data = int(do_data_text)
-        except ValueError:
-            msgbox = QMessageBox(QMessageBox.Warning, "Błąd", "Nieprawidłowa data początkowa.")
-            msgbox.exec_()
-            return False                  
+        def model3d_poprawnosc_dat(od_data_text, do_data_text):
+            current_year = datetime.now().year
+            min_year = 1970
```

Comment:
Proszę zmień nazwę tej zmiennej na bardziej adekwatną, np. min_year_buildings_3d i przypisz jako zmienna globalna, ponieważ też raz na daną sesję to moze być wykonane